src/components/account/selectedAccount.tsx

```
@@ -417,30 +414,46 @@ export const SelectedAccountInfo: React.FC<{}> = () => {
-                      }
-                    >
-                      <Stack paddingY={0.5} direction="row" alignItems="center" width="100%">
-                        <Box textAlign="left" flexGrow={1}>
-                          <Typography variant="subtitle2">{t.time.toLocaleString()}</Typography>
-                        </Box>
-                        <Box textAlign="right" width="35%">
-                          {t.amount} {t.symbol}
-                        </Box>
-                      </Stack>
-                    </Button>
+                    <Grid container direction={'row'} alignItems={'center'}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)